L1Trigger/L1TMuonEndCap/interface/bdt/LossFunctions.h

```diff
@@ -1,6 +1,6 @@
 // LossFunctions.h
```

Comment:
Would be helpful to grab the old version and just change the "virtual" lines ... ~20 lines with spacing changes, just 2 with something really different.